### DISCUSSION TITLE
CompileError: Cannot find import 'package:flutter_eval/flutter_eval.dart'

### DISCUSSION THREAD

**Discussion by nicktt19** [ethanblake4/flutter_eval#26] [General]
Hello, i am following the readme to build the example(0.5.3), and get a CompileError, thanks.

![image](https://user-images.githubusercontent.com/106294970/204702643-4a8a4024-5632-4fb2-a0a2-2facfe06313e.png)

**Comment by ethanblake4** [comment]
The problem is that you are importing `'package:flutter_eval/flutter_eval.dart'` inside the code you are _compiling with flutter\_eval_. Instead you should import `'package:flutter/flutter.dart'`.

**Reply by nicktt19** [reply]
i am compiling in the root of your example project. Evan if i replace the `'package:flutter_eval/flutter_eval.dart'` to `import 'package:flutter/flutter.dart'`. It shows same kind of error `CompileError: Cannot find import 'package:flutter/flutter.dart' (while parsing 'package:example/main.dart') at unknown
`

![image](https://user-images.githubusercontent.com/106294970/204706509-5aeb2a37-17a4-4be6-8a6a-d9633ee360a2.png)

**Comment by ethanblake4** [comment]
Oops I meant `'package:flutter/material.dart'`

**Comment by ethanblake4** [comment]
Atually, after looking closer I realize you are using the dart_eval CLI. The dart_eval CLI cannot compile dart_eval itself. 
- If you want to use the CLI, you should create a new blank Flutter project using `flutter create` and then run `dart_eval compile -o program.evc` in the root of that
- If you want to run the example project from the flutter_eval repo you can't use the dart_eval CLI, you have to just use `flutter run`
